app/assets/stylesheets/common/core.scss

```diff
@@ -439,6 +439,10 @@ th.required::after {
   vertical-align: middle;
 }
 
+.fa-download-o::before {
+  content: asset-url('icons/download.png');
```

Comment:
Okay, so it looks like we already have a `download-file.png` icon used in some places. But, the icon you added is a true icon size, which is good. You can go ahead and *replace* the existing `download-file.png` file with the icon you added, and then here use `icons/download-file.png` instead.

Reply:
Oh also, please modify the file manager "Download All" action to use this icon instead of the "Save disk" icon it currently uses.

Reply:
And one last thing: it would be good to add a hover effect to this icon as well, to distinguish it from the rest of the row when a user hovers over it. Maybe add a slight background colour or something, not sure.

Reply:
Using a slightly stronger color for the icon hover over (relative to the row's hover)

Reply:
Updated PR description with the new changes (changed download all icon, hover over background color change for both the row item & the download icon)